src/OmniSharp.Roslyn.CSharp/Services/Structure/CodeStructureService.cs

```diff
@@ -83,6 +83,9 @@ private IEnumerable<CodeElement> CreateCodeElements(SyntaxNode node, SourceText
                 case EnumDeclarationSyntax enumDeclaration:
                     yield return CreateCodeElement(enumDeclaration, text, semanticModel);
                     break;
+                case EnumMemberDeclarationSyntax enumMemberDeclarationSyntax:
```

Comment:
Thanks! I forgot that `EnumMemberDeclarationSyntax` doesn't inherit from `BaseFieldDeclarationSyntax`.

Reply:
Could you move this below the case for `BaseFieldDeclarationSyntax`?